docs/docs/concepts/how-it-works/spfx-project-upgrade.mdx

```diff
@@ -0,0 +1,227 @@
+# How it works: spfx project upgrade
```

Comment:
Let's add a front-matter title so it appears better in the sidebar navigation. That way, under the section `How it works`, it will show as `spfx project upgrade` instead of `How it works: spfx project upgrade`

```suggestion
---
title: spfx project upgrade
---

# How it works: spfx project upgrade
```